### ISSUE TITLE
Profile Issues

### ISSUE THREAD

**Issue by thelapisfox** [Nexus-Mods/Vortex#5035]
**Describe the bug**
I am not sure if this is a bug, if I am doing something wrong, or if this is even something that is possible. I was playing Fallout 4 on two different systems with two different setups. I decided it was time to move the profile from one system to the other. So, I copied all of the mods from the one and installed them on the other (installed not enabled) and then copied the profile itself. When I started up Vortex, it did not see the copied profile so I created a new profile and copied both loadorder.txt and plugins.txt from the copied profile into the new profile. I then went into Vortex and tried to enable the profile but it still says 0 mods. Looking at the copied files, it turns out there Vortex completely replaced them with mostly blank files. I do not see where in the file system it could be storing a backup of the profiles so I have no clue how to force Vortex to use MY copies of the profile files

**To Reproduce**
1. Copy profile from another install;
2. Paste profile into AppData/Roaming/Vortex/Fallout4/profiles
3. Attempt to use new profile

**Expected behavior**
Vortex should use the transferred plugins.txt and loadorder.txt files for it's profiles. It should not be using something else. It makes those files obsolete if the program just ignores and replaces them anyways.

**Screenshots**
Screenshots below shows you the issue. As shown in the screenshots, Vortex lists 1 mod (Raider Run profile) even though files list 217. Once Vortex is closed, files will be replaced with whatever it is that it is using.
![files](https://user-images.githubusercontent.com/899111/60761026-2fd80180-9ff5-11e9-993b-879b5558b24c.PNG)
![vortex](https://user-images.githubusercontent.com/899111/60761027-336b8880-9ff5-11e9-90f3-f961a13d9881.PNG)

**Platform (please complete the following information):**
 - OS: Windows 10
 - Vortex Version: 0.18.12 (updated today)

**Comment by IDCs** [comment]
Hi @thelapisfox, I assume you're trying to copy over the plugins and loadorder files as you've defined dependencies within Vortex and do not wish do go through creating them again ? Unfortunately there's no way to export/import dependencies between Vortex installations (yet) and therefore you will have to re-define those dependencies manually.

Regarding saving/transferring plugins/loadorder files, Vortex will always generate those files using the LOOT library; allowing users to do this manually was never part of our design.

Finally, the blank files - I'm wondering how you installed these mods; did you per-chance just copied them into Vortex's staging folder ? also, were you using the move deployment method on your other system ?

**Comment by TanninOne** [comment]
The primary "plugins.txt"/"loadorder.txt" for the current profile are always the ones in "%LOCALAPPDATA%\<game name>"
As long as the profile is active, changes will be synced from those primary files _to_ the profile, *not* the other way around.
So if you want to manually change these files for the active profile you edit those primary files like any tool unaware of Vortex would do.

As for the rest, this is simply not supported and won't be easily possible, Vortex stores information about installed mods, archives and profiles in a database that puts them into relation with each other, you can't just copy a bunch of files and expect Vortex to magically know which mod belongs to which profile.
We'd need a feature to export profiles into a format adequate to transfer it to a different system and import it there.
This would be a very nifty project for an extension, it's not currently a supported feature.

**Comment by thelapisfox** [comment]
I copied the plugins.txt and loadorder.txt from the "%appdata%/Vortex/gamename/profile" folder on the old system and pasted it into both the one above and "%localappdata%/gamename" folder on the new system. As for the mods, I copied both the download folder and the staging folders and transferred both. As for the two txt files, they are in the primary location on the new system. It seemed like, no matter what I did, Vortex erased those two files and would replace them with a different primary that it was storing somewhere else.

Though I do understand profile transferring not being a supported feature, but if you say that the files in the location above is suppose to be the primary files, then there is still an issue.

Also, I knew I would have to redo the mod rules I created when I transferred to the new system, I was just hoping that by transferring over those files, Vortex would at least auto enable the correct mods for me. Unfortunately, I ended up having to go through it one mod at a time and the load order is still completely different because it keeps overwriting the loadorder.txt file, no matter where I put it. Because the load order was so different, I was forced to start a whole new game because it ended up completely corrupting my save files all the way back to when I loaded up the game on the new system (instant ctd on save load).
